lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -1,24 +1,94 @@
-import * as React from 'react';
-import { DatePicker } from '../DatePicker';
+import React, { useState } from 'react';
+import frLocale from 'date-fns/locale/fr';
+import DateFnsUtils from '@date-io/date-fns';
+import enLocale from 'date-fns/locale/en-US';
+import Button from '@material-ui/core/Button';
 import TextField from '@material-ui/core/TextField';
-import { screen } from '@testing-library/react';
-import { createClientRender } from './createClientRender';
+import LocalizationProvider from '../LocalizationProvider';
+import { MaterialUiPickersDate } from '../typings';
```

Comment:
What's the public API variation of this type?

Reply:
```jsx
import { MaterialUIPickersDate } from '@material-ui/pickers'
```

Reply:
What about we reexport

```jsx
import { DateType } from '@date-io/type';
```

```suggestion
import { DateType } from '@material-ui/pickers'
```

and then:

```jsx
const [date, setDate] = useState<DateType | null>(new Date('01/01/2020'));
```

Reply:
Now, because the topic is broader than this pull request, I have opened an RFC, so we can handle the whole codebase #1962, but at least here, we have a great example for such a proposal.